cli/lib/nf3.mjs

```diff
@@ -786,12 +810,13 @@ class Nf3 {
   /**
   Set a Web3 Provider URL
   */
-  setWeb3Provider() {
+  async setWeb3Provider() {
     this.web3 = new Web3(this.web3WsUrl, { transactionBlockTimeout: 200 }); // set a longer timeout
     if (typeof window !== 'undefined') {
       if (window.ethereum && this.ethereumSigningKey === '') {
         this.web3 = new Web3(window.ethereum);
-        window.ethereum.request({ method: 'eth_accounts' });
+        await window.ethereum.enable();
```

Comment:
Deprecated function. Needs to be replaced with `ethereum.request({ method: 'eth_requestAccounts' })`

Reply:
You're right.